data/mships.txt

```diff
@@ -2925,8 +2925,8 @@ ship "CK80T Tlielart"
 	explode "large explosion" 40
 	explode "huge explosion" 10
 	"final explode" "final explosion large"
-	description `Massive freighter developed to carry as much cargo as possible, and did so exceeding the capacity of every other ship by far. Though unlike the other ship from MarStel Industries, it is very lightly armed relative to cargo capacity making it lacks the firepower to properly protect itself and neither did it have speed to dodge any raider. Due to such reason, it will require decent escort to safely travel.`
-	description `Though it carried more than most, if not all, freighter in human space the sheer amount of cargo that have to be unloaded will require ungodly amount of time, equipment, and manpower, which only get worse when loading cargo. Any less than well developed urban spaceport would struggle to service the ship and often will take a few days to complete the operation, if they ever require this much cargo in the first place. This results in the ship being not quite a popular choice in human space as few planets require this much cargo in a single delivery and fewer are capable of handling it.`
+	description `Massive freighter developed to carry as much cargo as possible, handily exceeding the capacity of its competitors the moment it was launched. However, unlike the other ships from MarStel Industries, it is slow, unmaneuverable, and very lightly armed relative to its cargo capacity, lacking enough armament to defend itself properly and enough speed to outrun any threats. Due to those reasons, it requires well-equipped escorts to travel safely, especially through dangerous territory.`
+	description `Though it carries more than most, if not all, freighters in human space, the sheer amount of cargo that has to be unloaded during a stop requires an ungodly amount of time, equipment, and manpower, which only gets worse when loading more cargo. Any less than well developed urban spaceport struggles to service the ship and often takes a few days to complete the operation, if they ever require this much cargo in the first place. This results in the ship being not quite a popular choice in human space as few planets require this much cargo in a single delivery and fewer are capable of handling it.`
```

Comment:
```suggestion
	description `Though it carries more than most, if not all, freighters in human space, the sheer amount of cargo that has to be unloaded during a stop requires an ungodly amount of time, equipment, and manpower. This only gets worse when loading cargo back on to the ship. Any less than well developed urban spaceport struggles to service the ship and often takes a few days to complete the operation, if they ever require this much cargo in the first place. This results in the ship being not quite a popular choice in human space as few planets require this much cargo in a single delivery and fewer are capable of handling it.`
```

Reply:
Yeah, you're right, that conveys it fine. I wasn't quite sure, so I just wrote what looked grammatically sound